frontend/src/components/Common/ExpandableTable.jsx

```diff
@@ -0,0 +1,105 @@
+import {
+  MaterialReactTable,
+  useMaterialReactTable,
+} from "material-react-table";
+import { Box, IconButton, Tooltip, useTheme } from "@mui/material";
+
+const ExpandableTable = ({
+  data,
+  columns,
+  actions,
+  subTableColumns,
+  subData,
+}) => {
+  const enableActions = actions;
+  const theme = useTheme();
+
+  const table = useMaterialReactTable({
+    columns: columns,
+    data: data,
+    enableExpanding: true,
+    enableExpandAll: false,
```

Comment:
It would be good to enable the option to expand all so the user could be able with just one click see all the data for the inner tables.

Reply:
I set it to true!